packages/card/src/Card.tsx

```diff
@@ -33,28 +33,18 @@ const lightHoverBoxShadow = `0 4px 20px -4px ${transparentize(
   0.8,
   palette.black,
 )}`;
-const lightFocusBoxShadow = `
-  0 0 0 2px ${palette.white},
-  0 0 0 4px ${palette.blue.light1}
-`;
-const darkBaseBoxShadow = `0 4px 20px -4px  ${transparentize(0.3, '#000')}`;
-const darkHoverBoxShadow = `0 2px 12px -2px ${transparentize(0.1, '#000')}`;
-const darkFocusBoxShadow = `0 0 0 3px ${uiColors.focus}`;
+const lightFocusBoxShadow = focusRing['light'].default;
```

Comment:
Small opinion - I feel like now that we have a token to reuse, we could just define this on line 48 instead of having a separate variable